vertx-mysql-client/src/main/java/io/vertx/mysqlclient/MySQLConnectOptions.java

```diff
@@ -145,6 +146,26 @@ public MySQLConnectOptions setSsl(boolean ssl) {
     return this;
   }
 
+  /**
+   * Set the server RSA public key which is mostly used for encrypting password under insecure connections when performing authentication.
+   *
+   * @param serverRsaPublicKey the value of the server RSA public key
+   * @return a reference to this, so the API can be used fluently
+   */
+  public MySQLConnectOptions setServerRsaPublicKey(String serverRsaPublicKey) {
```

Comment:
any key should either be a path or a Buffer like in `PemKeyCertOptions`

Reply:
updated in https://github.com/eclipse-vertx/vertx-sql-client/pull/420/commits/6225f912d3f6d37f8afee8fb030b851f0c4bf4e5.